Movie Challenge/Movie Challenge/Controllers/Home/Cells/SuggestionTableViewCell.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SuggestionTableViewCell.swift
+//  Movie Challenge
+//
+//  Created by Igor Gomes Arantes on 14/09/2018.
+//  Copyright © 2018 igor gomes arantes. All rights reserved.
+//
+
+import UIKit
+
+class SuggestionTableViewCell: UITableViewCell, UICollectionViewDelegate, UICollectionViewDataSource{
+    
+    @IBOutlet weak var categoryLabelView: UILabel!
+    @IBOutlet weak var suggestionMoviesCollectionView: UICollectionView!
+    @IBOutlet weak var suggestionHeaderView: UIView!
+    
+    private var moviePage = MoviePageDTO()
+    private var sort: Sort!
+    private var posters = [UIImage]()
+    private var getPosterTasks: [URLSessionDataTask]!
+    
+    override func awakeFromNib() {
+        getPosterTasks = [URLSessionDataTask]()
```

Comment:
You forgot to call the super method, be careful with this because it could cause serious problems. (In most cases your application could crash and you will never know where the app crash)